tycho-baseline-plugin/src/main/java/org/eclipse/tycho/baseline/FeatureBaselineComparator.java

```diff
@@ -334,7 +334,7 @@ private List<Diff> computeJarDelta(MavenProject project, BaselineContext context
 			ignores.addAll(context.getIgnores());
 			ArtifactDelta artifactDelta = zipComparator.getDelta(getStream(baselineJarUnit, context),
 					new ComparatorInputStream(reactor, file.getAbsolutePath()),
-					new ComparisonData(ignores, false));
+					new ComparisonData(ignores, List.of(".xml"), false));
```

Comment:
I don't think we should ignore any xml here this does not seem correct.

Reply:
It is not about ignoring, it is about defining if a (feature)XML-file is compared as text-file or not.
And I would say that a feature.xml should be compared as text-file, if not compared per XML-element.